pom.xml

```diff
@@ -76,7 +75,33 @@
     <dependency>
       <groupId>life.qbic</groupId>
       <artifactId>data-model-lib</artifactId>
-      <version>1.8.0</version>
+      <version>1.9.0-SNAPSHOT</version>
```

Comment:
Is it possible to use `1.8.3` instead? Snapshot versions in the pom are dangerous.

Reply:
That's a good point! Will fix it, Thank you :)